### PR TITLE
Adjust guns

### PR DESCRIPTION
**Pull request by toberge**
- Slug barrel isn't implemented, it's simply a clone of the bullet barrel. **Removed**. 
- Double barrel is broken at the moment, and makes no sense as a barrel with our current design (should be an extension). **Removed**. 
- Bullet range was too low, so I doubled it :smiling_imp: 

Closing #160

### CODE REVIEW

File: Assets/Scripts/Augment/AugmentImplementations/ContinuousDamage.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    private void OnDeath(HealthController healthController, float damage, DamageInfo info)
+    {
+        this.enabled = false;
+    }
+
+    private void Update()
+    {
+        if (hitbox && Time.time - lastDamageTime > 1 / damageRate)
+        {
```
Comment (Fueredoriku) [inline Assets/Scripts/Augment/AugmentImplementations/ContinuousDamage.cs:35]: We might want to add a onHit invocation here, but it's not strictly necessary yet

File: Assets/Scripts/Extensions/TransformExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+using UnityEngine;
+
+namespace TransformExtensions
```
Comment (Fueredoriku) [inline Assets/Scripts/Extensions/TransformExtensions.cs:3]: Nice use of type extension!

**Review by Fueredoriku — APPROVED** [review]
Great fixes/changes!

Added a small commit to fix a bug where players would keep taking continous damage after death.

I would suggest we try to do something about the instatiated fire scale later, but it doesn't need to be in in this PR.

I'll leave an approve here, good work! 😎👍

File: Assets/Scripts/Extensions/TransformExtensions.cs (hunk 2 above)
Comment (toberge) [reply Assets/Scripts/Extensions/TransformExtensions.cs:3]: Thanks!

File: Assets/Scripts/Augment/AugmentImplementations/ContinuousDamage.cs (hunk 1 above)
Comment (toberge) [reply Assets/Scripts/Augment/AugmentImplementations/ContinuousDamage.cs:35]: That's a little outside the aim of this component, but it seems like the sort of thing we would want for our _actual_ hat barrel implementation

**Pull request merged**